README.md

```diff
@@ -135,21 +192,28 @@ Released under the BSD-3-Clause License. See `LICENSE` for more information.
 - [Gradle](https://gradle.org) under [Apache v2](https://github.com/gradle/gradle?tab=Apache-2.0-1-ov-file#readme).
-- [HTTP Components](https://hc.apache.org/httpcomponents-client-5.4.x/index.html) under [Apache v2](https://www.apache.org/licenses/).
-- [Jetty](https://jetty.org/index.html) under [Apache v2](https://jetty.org/docs/jetty/12/index.html).
+- [JTS Topology Suite](https://locationtech.github.io/jts/) for geometry handling under [EDL](https://www.eclipse.org/org/documents/edl-v10.php).
 - [JUnit](https://junit.org/junit5/) under [EPL v2](https://github.com/junit-team/junit5).
+- [Math.js](https://mathjs.org/) for mathematical expressions under [Apache v2](https://github.com/josdejong/mathjs/blob/develop/LICENSE).
 - [Minio Java SDK](https://min.io/docs/minio/linux/developers/java/minio-java.html) under [Apache v2](https://github.com/minio/minio-java?tab=Apache-2.0-1-ov-file#readme).
+- [Mockito](https://site.mockito.org/) for testing under [MIT](https://github.com/mockito/mockito/blob/main/LICENSE).
+- [Picocli](https://picocli.info/) for command line parsing under [Apache v2](https://github.com/remkop/picocli/blob/main/LICENSE).
+- [Popper.js](https://popper.js.org/) for tooltip positioning under [MIT](https://github.com/floating-ui/floating-ui/blob/master/LICENSE).
 - [Public Sans](https://public-sans.digital.gov/) font under [OFL-1.1](https://github.com/uswds/public-sans/blob/master/LICENSE.md).
-- [QUnit](https://qunitjs.com/) for unit front-end testing under [MIT](https://github.com/qunitjs/qunit/blob/main/LICENSE.txt).
-- [SVG Spinners](https://github.com/n3r4zzurr0/svg-spinners?tab=readme-ov-file) under [MIT](https://github.com/n3r4zzurr0/svg-spinners?tab=readme-ov-file)
+- [SLF4J](https://www.slf4j.org/) for logging under [MIT](https://github.com/qos-ch/slf4j/blob/master/LICENSE.txt).
+- [Spotless](https://github.com/diffplug/spotless) for code formatting under [Apache v2](https://github.com/diffplug/spotless/blob/main/LICENSE).
 - [Tabby](https://github.com/cferdinandi/tabby) for tab interface management under [MIT](https://github.com/cferdinandi/tabby/blob/master/LICENSE.md).
-- [Webpack](https://webpack.js.org/) for bundling JavaScript modules under [MIT](https://github.com/webpack/webpack/blob/main/LICENSE).
+- [TeaVM](https://teavm.org/) for WebAssembly compilation under [Apache v2](https://github.com/konsoletyper/teavm/blob/master/LICENSE).
+- [Tippy.js](https://atomiks.github.io/tippyjs/) for tooltips under [MIT](https://github.com/atomiks/tippyjs/blob/master/LICENSE).
+- [UCAR NetCDF](https://www.unidata.ucar.edu/software/netcdf-java/) for NetCDF support under [BSD-3](https://github.com/Unidata/netcdf-java/blob/master/LICENSE).
+- [Undertow](https://undertow.io/) for the local web server under [Apache v2](https://github.com/undertow-io/undertow/blob/master/LICENSE.txt).
 
 Also uses [Tumerin]().
```

Comment:
Please say that we recommend Tumerin with link to https://projects.eclipse.org/projects/adoptium.temurin